pandas/tests/groupby/test_value_counts.py

```diff
@@ -79,3 +79,28 @@ def rebuild_index(df):
     # have to sort on index because of unstable sort on values
     left, right = map(rebuild_index, (left, right))  # xref GH9212
     tm.assert_series_equal(left.sort_index(), right.sort_index())
+
+
+@pytest.mark.parametrize("freq", ["1D", "2D", "1W", "1Y"])
+@pytest.mark.parametrize("size", [100, 1000])
+@pytest.mark.parametrize("frac", [0.1, 0.5, 1])
+def test_series_groupby_value_counts_with_grouper(freq, size, frac):
+    np.random.seed(42)
+
+    df = DataFrame.from_dict(
+        {
+            "date": date_range("2019-09-25", periods=size),
+            "name": np.random.choice(list("abcd"), size),
+        }
+    ).sample(frac=frac)
+
+    gr = df.groupby(Grouper(key="date", freq=freq))["name"]
+
+    # have to sort on index because of unstable sort on values xref GH9212
+    result = gr.value_counts().sort_index()
+    expected = gr.apply(Series.value_counts).sort_index()
+    expected.index.names = (
+        result.index.names
+    )  # .apply(Series.value_counts) can't create all names
```

Comment:
I don't understand the comment here. Can it be removed?

Reply:
Oh no! thx, it's just my mistake... I'll remove it